src/mds/MDSMap.cc

```diff
@@ -785,6 +787,7 @@ void MDSMap::encode(bufferlist& bl, uint64_t features) const
     encode(min_compat_client, bl);
   }
   encode(required_client_features, bl);
+  encode(bal_rank_mask, bl);
```

Comment:
Please bump encoder version to 17 above.

Reply:
The version is set to 17. Please take a look at modifications?